src/components/Book.js

```diff
@@ -0,0 +1,25 @@
+import PropTypes from 'prop-types';
+
+const Book = ({ book }) => (
+  <li>
+    <div className="bookContainer">
+      <h2>{book.Title}</h2>
+      <p>{book.Author}</p>
+      <div className="actionsContainer">
+        <button type="button">Comment</button>
+        <button type="button">Comment</button>
+        <button type="button">Comment</button>
+      </div>
+    </div>
+  </li>
+);
```

Comment:
- [ ] Please make sure to add a delete button for books as requested in the project requirements.

![image](https://github.com/lorenzoworx/bookstore-react/assets/83803414/bb253d26-33f6-4129-a9a3-db8034762399)